lib/prawn/font.rb

```diff
@@ -68,6 +71,51 @@ def font(name = nil, options = DEFAULT_OPTS)
       @font
     end
 
+    def font_family
+      @font ? @font.family : 'Helvetica'
+    end
+
+    # @method font_style(points=nil)
+    #
+    # When called with no argument, returns the current font style.
+    #
+    # When called with a single argument but no block, sets the current font
+    # style.  When a block is used, the font style is applied transactionally and
+    # is rolled back when the block exits.  You may still change the font style
+    # within a transactional block for individual text segments, or nested calls
+    # to font_style.
+    #
+    #   Prawn::Document.generate("font_style.pdf") do
+    #     font_style :bold
+    #     text "Bold text"
+    #
+    #     font_style(:italic) do
+    #       text "Italic text"
+    #       text "Bold text", :style => :bold
+    #       text "Italic text"
+    #     end
+    #
+    #     text "Bold text"
+    #   end
+    #
+    # Font styles are not additive, i.e. if the current style is :bold, setting
+    # the style to :italic results in italic, not bold-italic text. Use
+    # :bold_italic instead.
+    #
+    # When called without an argument, this method returns the current font
+    # style.
+    #
+    def font_style(style = nil, &block)
+      return @font ? @font.style : :normal if style.nil?
+
+      font(font.name, style: style, &block)
```

Comment:
Reading the associated tests I think the tests might work because `name` and `family` are the same for 'Helvetica'. Could you add a test that uses a registered TrueType font?